lambdas/preview/index.py

```diff
@@ -34,15 +36,17 @@
 def lambda_handler(params, _):
     """
     dynamically handle preview requests for bytes in S3
-
     caller must specify input_type (since there may be no file extension)
+
+    Returns:
+        JSON response
     """
     url = params['url']
     input_type = params.get('input')
 
     resp = requests.get(url)
     if resp.ok:
-        with NamedTemporaryFile() as file_:
+        with NamedTemporaryFile(mode='r+b') as file_:
```

Comment:
Does `'r+b'` make a difference?

Reply:
oh you're right. it _should_ but it doesn't even work.